oat-sa/extension-tao-itemqti-pci
# fix/AUT-2008/caret-math-entry-aligned-to-left

**Related to:** [AUT-2008](https://oat-sa.atlassian.net/browse/AUT-2008)

**Description:**
The caret for the Math interaction is aligned to the right but mathematical formulas are displayed in LTR even when we are in an RTL item so the caret should be aligned to the left in this case.

**Changes:**

- Remove RTL CSS rule and set the caret to align to the left.

**How to check:**
- if you using existing instance, you will need to run migration `sudo -u www-data php index.php '\oat\tao\scripts\tools\Migrations' -c migrate -v 'oat\qtiItemPci\migrations\Version202203221239221465_qtiItemPci' -e qtiItemPci`
- Create an item with RTL language.
- Add a Math Entry PCI.
- Go to the Response section of the interaction
- Click on Add answer option and click on edit the answer to access to see the caret.

![image](https://user-images.githubusercontent.com/11692751/159485973-e73de826-b7d0-4ba7-8ef1-781c07aee974.png)
